string  joint_name
float32 joint_position
float32 joint_target
float32 joint_torque
float32 joint_temperature
float32 joint_current 
string  error_flag
